lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -219,7 +220,7 @@ class ReportUIFeatures {
    * @return {Map<number, HTMLTableRowElement>}
    */
   _getThirdPartyRows(el, finalUrl) {
```

Comment:
maybe we should just pass the url items in then since it's happening the line before?